### PR TITLE
Pathology performance notebook

### PR DESCRIPTION
**Pull request by drbeh**
### Description
This PR adds notebook and all the other required codes for profiling pathology metastasis detection pipeline.

### Status
**Ready**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

**Comment by drbeh** [comment]
Thanks Ziyue for helping to make this notebook and running experiments.

File: acceleration/fast_model_training_guide.md [hunk 1]
```diff
@@ -248,3 +259,9 @@ In summary, with all the optimization, the training time of 8 V100 GPUs to achie
 ![brats benchmark](../figures/brats_benchmark.png)
 
 More details is available at [Brats distributed training tutorial](https://github.com/Project-MONAI/tutorials/blob/master/acceleration/distributed_training/brats_training_ddp.py).
+
+
+### 3. Pathology metastasis detection task
+
+- Algorithm experiments based on dataset analysis. (1) Training dataset too large to effectively utilize caching mechanisms. (2) Fast convergence within a few epochs, and simple network (Resnet18), such that there is little room for performance gain from training perspective. (3) The major bottleneck comes from data transformations, especially ColorJitter which is a random permutation of communication of four transforms.
+- We employed CuCIM IO for loading whole slide images and cuCIM transforms for performing the chain of transforms on GPU. In this way, we accelerating the data loading, data augmentation and preprocessing. We are getting 3.8X speedup on DGXA100. The chart below shows how it is compared to the baseline.
```
Comment (Nic-Ma) [inline acceleration/fast_model_training_guide.md:267]: Could you please help add the link to the Pathology tutorial?
And also add some charts to show more details.

Thanks.

File: acceleration/fast_model_training_guide.md [hunk 2]
```diff
@@ -222,6 +222,17 @@ More details about the PyTorch distributed training setup, please refer to: http
 We obtained U-Net performance benchmarks of Brain tumor segmentation task for reference (based on CUDA 11, NVIDIA V100 GPUs):
 ![distributed training results](../figures/distributed_training.png)
 
+## Leveraging specialized GPU transforms for digital pathology
```
Comment (Nic-Ma) [inline acceleration/fast_model_training_guide.md:225]: I would suggest move this section to `Optimizing GPU` part.

Thanks.

**Review by Nic-Ma — COMMENTED** [review]
Hi @wyli ,

Could you please help also review this PR?

Thanks.

**Review by wyli — APPROVED** [review]

**Pull request merged**